storage/oasis/client.go

```diff
@@ -49,8 +68,24 @@ func NewRuntimeClient(ctx context.Context, sourceConfig *config.SourceConfig, ru
 		return nil, err
 	}
 
+	// todo: short circuit if using purely a file-based backend and avoid connecting
```

Comment:
Do you plan this for this PR still? Looks like it should be very similar to the solution in `NewConsensusClient`.

Fine to leave it for a later PR (likely the one that integrates into CI) too.

Reply:
I'd prefer to leave it for a future PR; storing the `runtimeInfo` offline requires additional config changes, and I'd like to do more digging into why we fetch the info from the node and how it's used.

Reply:
Ah thank you, I overlooked the `info`. And you even mentioned it in a previous comment IIRC, but I didn't make the connection. Yeah SG to leave for a future PR.